app/src/main/res/values/strings.xml

```diff
@@ -195,7 +195,7 @@
     <string name="label_menu_settings">Settings</string>
     <string name="label_menu_exit">Exit</string>
 
-    <string name="permission_dialog_setting">settings</string>
+    <string name="permission_handler_permission_dialog_setting">settings</string>
```

Comment:
Should we remove this string definition since it's already defined in `modules/permissionhandler/src/main/res/values/strings.xml`?

Reply:
We want to keep it so it'll be localized by the l10n team